assets/js/blocks/cart-checkout/checkout/block.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "woocommerce/checkout",
+  "version": "1.0.0",
+  "title": "Checkout",
+  "description": "Display a checkout form so your customers can submit orders.",
+  "category": "woocommerce",
+  "keywords": [ "WooCommerce" ],
+  "supports": {
+	"align": [ "wide" ],
+	"html": false,
+	"multiple": false
+  },
+  "attributes": {
+	"isPreview": {
```

Comment:
Not sure if this is just the preview, but it seems this JSON file lacks indenting.

Reply:
Mine indented with spaces instead of tabs, I've updated it now